File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get user input on height of pyramid
    int n;
    do
    {
        n = get_int("Level of Pyramid: ");
    }
    while (n < 1 || n > 8);
    
    // Iterate Row
    for (int i = 0; i < n; i++)
    {
        // Iterate column
        for (int j = 0; j < n; j++)
        {
            // Check whether to output # or blank
            if ((n - 1 - j) <= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
    
}

File: Week 6/pset6/mario/mario.py
from cs50 import get_int

# User input
while True:
    number = get_int("Number: ")

    if number >= 1 and number <= 8:
        break

# Print
for i in range(number):
    print((number - i - 1) * " " + (i + 1) * "#")